nl/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -267,31 +267,31 @@ sendAnyway=Toch verzenden
+12595=Het lijkt erop dat de SMTP server %S geen versleutelde wachtwoorden ondersteunt. Probeer, als u dit account net heeft ingesteld, te wijzigen naar 'Normaal wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'. Neem, als het eerst wel werkte en nu opeens niet meer, contact op met uw e-mail administrator of provider.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 # LOCALIZATION NOTE (12596): $S is server hostname
-12596=The SMTP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+12596=De SMTP server %S staat geen wachtwoorden in platte tekst toe. Probeer te wijzigen naar 'Versleuteld wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'.
 
 ## @name NS_ERROR_SMTP_AUTH_FAILURE
 # LOCALIZATION NOTE (12597): $S is server hostname
-12597=Unable to authenticate to SMTP server %S. Please check the password, and verify the 'Authentication method' in 'Account Settings | Outgoing server (SMTP)'.
+12597=Niet is staat om te authenticeren naar SMTP server %S. Controleer alstublieft het wachtwoord en verifieer de 'Authenticatiemethode' in de 'Account Instellingen | Uitgaande server (SMTP)'.
 
 ## @name NS_ERROR_SMTP_AUTH_GSSAPI
 # LOCALIZATION NOTE (12598): $S is server hostname
 12598=Het Kerberos/GSSAPI ticket werd niet geaccepteerd door de SMTP server %S. Controleer a.u.b. of u ingelogd bent op de Kerberos/GSSAPI realm.
 
 ## @name NS_ERROR_SMTP_AUTH_MECH_NOT_SUPPORTED
 # LOCALIZATION NOTE (12599): $S is server hostname
-12599=The SMTP server %S does not support the selected authentication method. Please change the 'Authentication method' in the 'Account Settings | Outgoing Server (SMTP)'.
+12599=De SMTP server %S ondersteunt de geselecteerde Authenticatiemethode niet. Wijzig alstublieft de 'Authenticatiemethode' in de 'Account Instellingen | Uitgaande Server (SMTP)'.
 
 ## @name NS_ERROR_SMTP_AUTH_NOT_SUPPORTED
 # LOCALIZATION NOTE (12600): $S is server hostname
-12600=Unable to authenticate to SMTP server %S. It does not support authentication (SMTP-AUTH) but you have chosen to use authentication. Please change the 'Authentication method' to 'None' in the 'Account Settings | Outgoing Server (SMTP)' or contact your email service provider for instructions.
+12600=Niet is staat om te authenticeren naar SMTP server %S. Hij ondersteunt authenticatie (SMTP-AUTH) niet, maar u heeft ervoor gekozen om authenticatie te gebruiken. Wijzig alstublieft de 'Authenticatiemethode' naar 'Geen' in de 'Account Instellingen | Uitgaande Server (SMTP)' of neem contact op met uw e-mailserviceprovider voor instructies.
```

Comment:
CORRECTION NEEDED, it should be: '' Niet in staat om te authenticeren naar SMTP server %S. Authenticatie (SMTP-AUTH) wordt niet ondersteund, maar u heeft ervoor gekozen om authenticatie te gebruiken. Wijzig alstublieft de 'Authenticatiemethode' naar 'Geen' in de 'Account Instellingen | Uitgaande Server (SMTP)' of neem contact op met uw e-mailserviceprovider voor instructies. ''

- typo: ''Niet in staat'' vs. ''Niet is staat''
- consider ''wordt ondersteund'' vs ''hij ondersteunt''